<launch>

  <node pkg="CylinderDetection" type="CylinderDetection" name="CylinderDetection" output="screen">
    launch-prefix="xterm -e gdb --args"
    <param name="base_frame" value="/bubbleRob" />
    <param name="max_range" value="2.0" />
    <param name="n_samples" value="300" />
    <param name="tolerance" value="0.02" />
    <param name="parse_plane_lower_bound_threshhold" value="10" />
    <param name="parse_plane_upper_bound_threshhold" value="600" />
    <param name="accept_cylinder_threshhold" value="10" />
    
    <remap from="/CylinderDetection/scans" to="/vrep/depthSensor"/>
  </node>
</launch>
